service/app/models/book.rb

```diff
@@ -0,0 +1,20 @@
+class Book < ActiveRecord::Base
+	include ApplicationModel
+
+	belongs_to :publisher, foreign_key: :publisher_id
+	has_many   :book_spines
+
+	validates :publisher_id, presence: true
+	validates :has_licensing, inclusion: { in: [0, 1] }, presence: true
+	validates :isbn, length: { is: 13 }, numeric_literality: true, uniqueness: {scope: :nbn}, presence: true
```

Comment:
__[rubocop]__
* Space inside { missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Line is too long. [105/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)
* Space inside } missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)